### PR TITLE
 [Android] Adjust image decoding when using indirect decoding size

### LINKED ISSUE #2368: [Android] Late-bound targetsize local image may not load
## Current behavior

Late-bound targetsize image may not load

## How to reproduce it (as minimally and precisely as possible)

https://dev.azure.com/nventive/Umbrella/_workitems/edit/170404

## Environment

Nuget Package:

Package Version(s):

Affected platform(s):

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renderers for Xamarin.Forms
- [ ] Windows
- [ ] Build tasks
- [ ] Solution Templates

Visual Studio:

- [ ] 2017 (version: )
- [ ] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins:

- [ ] Resharper (version: )

### PR DESCRIPTION
**Pull request by jeromelaban**
GitHub Issue (If applicable): Fixes #2368

## PR Type

What kind of change does this PR introduce?
- Bugfix

## What is the new behavior?

Adjust the decoding passes to tentatively use the measure available size as a decode size, in case the image does not have its own set of size constraint.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] Tested code with current [supported SDKs](https://github.com/unoplatform/uno/blob/master/README.md#uno-features)
- [x] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [x] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [x] [Wasm UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md#running-the-webassembly-ui-tests-snapshots) are not showing unexpected any differences. Validate PR `Screenshots Compare Test Run` results.
- [x] Contains **NO** breaking changes
- [x] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [x] Associated with an issue (GitHub or internal)

## Other information

Internal Issue (If applicable):
https://dev.azure.com/nventive/Umbrella/_workitems/edit/170404

### CODE REVIEW

**Comment by jeromelaban** [comment]
Re-running a CI, there are some tests that were in invalid state.

**Comment by carldebilly** [comment]
Seems better :-)

File: src/Uno.UI/UI/Xaml/Controls/Image/Image.Android.cs [hunk 1]
```diff
@@ -386,7 +436,7 @@ private async Task SetSourceUriOrStream(ImageSource newImageSource)
 
 				_imageFetchDisposable.Disposable = disposable;
 
-				var bitmap = await newImageSource.Open(disposable.Token, this, _targetWidth, _targetHeight);
+                var bitmap = await newImageSource.Open(disposable.Token, this, _targetWidth, _targetHeight);
 
 				if (newImageSource.IsImageLoadedToUiDirectly)
```
Comment (carldebilly) [inline src/Uno.UI/UI/Xaml/Controls/Image/Image.Android.cs]: Strange indentation ;-)

**Pull request merged**